malevo/swagger_v2.yml

```diff
@@ -0,0 +1,169 @@
+swagger: "2.0"
+info:
+  description: "This is the initial API proposal for the ConfusionFlow REST API"
+  version: "0.0.1"
+  title: "ConfusionFlow"
+  termsOfService: ""
+  contact:
+    email: "gfrogat@gmail.com"
+host: "localhost:8080"
+basePath: "/v2"
+tags:
+- name: "run"
+  description: ""
+- name: "view"
+  description: ""
+schemes:
+- "https"
+- "http"
+paths:
+  /run:
+    get:
+      tags:
+      - "run"
+      summary: "Show all available runs"
+      description: ""
+      operationId: "getRuns"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            type: "array"
+            items:
+              $ref: "#/definitions/Run"
+  /run/{runId}:
+    get:
+      tags:
+      - "run"
+      summary: "Get run by ID"
+      description: "Returns a single run"
+      operationId: "getRunById"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "runId"
+        in: "path"
+        description: "ID of run to return"
+        required: true
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Run"
+        400:
+          description: "Invalid ID supplied"
+        404:
+          description: "Run not found"
+  /view:
```

Comment:
Why not call it `views` since it returns all available views?

Reply:
missed that, sorry